Code/GraphMol/MolStandardize/Wrap/Tautomer.cpp

```diff
@@ -268,6 +268,18 @@ PyTautomerEnumeratorResult *enumerateHelper(
   return new PyTautomerEnumeratorResult(self.enumerate(mol));
 }
 
+std::vector<MolStandardize::TautomerScoringFunctions::SubstructTerm> GetDefaultTautomerSubstructsHelper() {
```

Comment:
why do you need this function?

Reply:
The return is a const vector, this protects that.  I.e. you can’t change the original vector.